client/me/security-2fa-backup-codes/index.jsx

```diff
@@ -158,6 +159,11 @@ class Security2faBackupCodes extends React.Component {
 	}
 }
 
-export default connect( null, {
-	recordGoogleEvent,
-} )( localize( Security2faBackupCodes ) );
+export default connect(
+	( state ) => ( {
+		printed: getUserSetting( state, 'two_step_backup_codes_printed' ),
```

Comment:
Nit: `printed` feels like it's a bit too broad. I'd make `printed` a bit more explanatory, like `backupCodesPrinted` or something. No strong feelings though.

Reply:
fixed via e662dd7